Doc/whatsnew/3.7.rst

```diff
@@ -446,35 +476,304 @@ Other Language Changes
+asynchronous context managers::
+
+    srv = await loop.create_server(...)
+
+    async with srv:
+        # some code
+
+    # At this point, srv is closed and no longer accepts new connections.
+
+(Contributed by Yury Selivanov in :issue:`32662`.)
+
+Callback objects returned by
+:func:`loop.call_later() <asyncio.AbstractEventLoop.call_later>`
+gained the new :meth:`when() <asyncio.TimerHandle.when>` method which
+returns an absolute scheduled callback timestamp.
+(Contributed by Andrew Svetlov in :issue:`32741`.)
+
+TCP sockets on Linux are now created with ``TCP_NODELAY`` flag set by default.
+(Contributed by Yury Selivanov and Victor Stinner in :issue:`27456`.)
+
+The :meth:`loop.create_datagram_endpoint() \
+<asyncio.AbstractEventLoop.create_datagram_endpoint>` method
+gained support for Unix sockets.
+(Contributed by Quentin Dawans in :issue:`31245`.)
+
+The :meth:`loop.create_connection() <asyncio.AbstractEventLoop.create_connection>`,
+:meth:`loop.create_server() <asyncio.AbstractEventLoop.create_server>`,
+:meth:`loop.create_unix_server() <asyncio.AbstractEventLoop.create_unix_server>`, and
+:meth:`loop.create_accepted_socket() <asyncio.BaseEventLoop.connect_accepted_socket>`
+now accept the *ssl_handshake_timeout* keyword argument.
+(Contributed by Neil Aspinall in :issue:`29970`.)
+
+The new :meth:`Handle.cancelled() <asyncio.Handle.cancelled>` method returns
+``True`` if the callback was cancelled.
+(Contributed by Marat Sharafutdinov. in :issue:`31943`.)
+
+The :meth:`loop.create_unix_connection() <asyncio.AbstractEventLoop.create_unix_connection>`
+method now supports now supports passing a :term:`path-like objects <path-like object>`
+as a socket path.  (Contributed by Yury Selivanov in :issue:`32066`.)
+
+The :func:`asyncio.windows_utils.socketpair`` function has been removed,
+:func:`socket.socketpair` should be used instead.
+(Contributed by Victor Stinner in :issue:`32154`.)
+
+The asyncio source has been converted to use the
+:keyword:`async`/:keyword:`await` syntax.
+(Contributed by Andrew Svetlov in :issue:`32193`.)
+
+Support for directly ``await``-ing instances of :class:`asyncio.Lock` and
+other asyncio synchronization primitives has been deprecated.  An
+asynchronous context manager has to be used in order to acquire and release
+the synchronization resource.  See :ref:`async-with-locks` for more
+information.
+(Contributed by Andrew Svetlov in :issue:`32253`.)
+
+The :meth:`ReadTransport.resume_reading() <asyncio.ReadTransport.resume_reading>`
+and :meth:`ReadTransport.pause_reading() <asyncio.ReadTransport.pause_reading>`
+are now idempotent.
+The new :meth:`ReadTransport.is_reading() <asyncio.ReadTransport.is_reading>`
+method can be used to determine the reading state of the transport.
+(Contributed by Yury Selivanov in :issue:`32356`.)
+
+Exceptions occurring in cancelled tasks are no longer logged.
+(Contributed by Yury Selivanov in :issue:`30508`.)
+
+
+.. _whatsnew37-asyncio-optimizations:
+
+Optimizations
```

Comment:
This is a list of modules, this section is misplaced.

Reply:
Again, asyncio is important enough to let people know about changes in it.